test/fixtures/es-module-loaders/register-multiple-programmatically-loaders.mjs

```diff
@@ -0,0 +1,10 @@
+import * as fixtures from '../../common/fixtures.mjs';
+import { registerLoader } from 'node:module';
+
+registerLoader(fixtures.path('es-module-loaders', 'loader-load-passthru.mjs'))
+await import(
```

Comment:
Ah, interesting: I had not thought to cover this case, but it is good to cover.

Reply:
Actually, that was your idea, or maybe I misunderstood 🤣

At this [comment](https://github.com/nodejs/node/pull/46826#discussion_r1194412298), where you talk about creating a test case registering multiple loaders across multiple files.